### DISCUSSION TITLE
Eu12 doesn’t work

### DISCUSSION THREAD

**Discussion by Philipock** [the-marenga/sf-scrapbook-helper#73] [General]
I can login to my account but it shows me no enemies.

**Comment by the-marenga** [comment]
eu12 does not have a pre-generated list of players, because that list would be out of date immediately

that means you have to crawl the server characters youself, which you can do by increasing the "Threads" value at the bottom left to a value > 0

The tool will then start to download all the characters in the Hall of Fame manually

**Comment by Philipock** [comment]
Thank you very muchGesendet mit der mobilen Mail AppAm 25/05/2024 um 13:48 schrieb Marenga
                
            
            
                
                    Von: "Marenga" ***@***.***>Datum: 25. Mai 2024An: "the-marenga/sf-scrapbook-helper" ***@***.***>Cc: "Author" ***@***.***>,"Philipock" ***@***.***>Betreff: Re: [the-marenga/sf-scrapbook-helper] Eu12 doesn’t work (Discussion #73)
                
                
eu12 does not have a pre-generated list of players, because that list would be out of date immediately
that means you have to crawl the server characters youself, which you can do by increasing the "Threads" value at the bottom left to a value > 0
The tool will then start to download all the characters in the Hall of Fame manually
